src/ShelfLayoutManager.Infrastructure/LaneEntity.cs

```diff
@@ -0,0 +1,18 @@
+namespace ShelfLayoutManager.Infrastructure;
+
+public record class LaneEntity
+{
+    public long Id { get; set; }
+
+    public long Number { get; set; }
+
+    public RowEntity? Row { get; set; }
+
+    public long? RowId { get; set; }
```

Comment:
Why is RowId nullable?
I am pretty sure a lane cannot exist without a row.

Reply:
EntityFramework is a bit weird, especially with relations.
Cardinality, cascading deletes, etc.

Easier to start with nullable thing and make it non-nullable over time.
Harder to go from non-nullable to nullable? Hmm. 🤔